docker/prowbazel/Makefile

```diff
@@ -1,5 +1,5 @@
 PROJECT = istio-testing
-VERSION ?= 0.5.7
+VERSION ?= 0.5.8
```

Comment:
Why is this a patch update?

Reply:
Why wouldn't it be? (genuine question) There doesn't seem to be any pattern for minor and patch updates, as far as I can tell. See: [docker/prowbazel/README.md](https://github.com/istio/test-infra/blob/master/docker/prowbazel/README.md).

Having said that, I've now noticed that this file should be updated in https://github.com/istio/test-infra/pull/1493, since image recipes are updated there and not this PR. Let me fix that.